docs/DEBUGGING.md

```diff
@@ -17,10 +19,24 @@ Default values launch.json for ESP-IDF Debug Adapter:
 
 Configuration settings of the ESP-IDF Debug Adapter for launch.json are:
 
+- `appOffset`: Program start address offset to start debugging.
 - `debugPort`: Port for ESP-IDF Debug Adapter. Default: 43474.
+- `env`: Environment variables to apply to the ESP-IDF Debug Adapter.
+- `gdbinitFile`: Specify the gdbinit file to send to gdb.
+- `initGdbCommands`: One or more xtensa-esp32-elf-gdb commands to execute in order to setup the underlying debugger. If gdbinitFile is defined, these commands will be ignored.
 - `logLevel`: Debug Adapter Debug level (0-4), 5 - for a full OOCD log. Default: 2.
 - `mode`: Can be either `auto`, to start the debug adapter and openOCD server within the extension or `manual`, to connect to existing debug adapter and openOCD session.
-- `env`: Environment variables to apply to the ESP-IDF Debug Adapter.
+
+> **NOTE:** As shown in [JTAG Debugging debugging with command line](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/jtag-debugging/using-debugger.html#command-line) make sure to include the following commands in gdbinit file or initGdbCommands, if used, for the debug session to properly work.
```

Comment:
@brianignacio5 Wouldn't it make sense to just have a fully defined `idf.initGdbCommands` json entry for the user to copy/paste or enable?

Reply:
Hm, nevermind, seems to be generated at the project level already?:

```
target remote :3333
symbol-file /Users/romanvg/dev/esp32s2_can/twai_network_slave/build/twai_network_slave.elf
mon reset halt
flushregs
thb app_main
c
```

Reply:
If one doesn't specify initGdbCommands or gdbinitFile, the file is generated from https://github.com/espressif/esp-debug-adapter/blob/master/debug_adapter/debug_backend/debug_backend/hw_specific/esp_init.gdb replacing with the actual .elf file

Will try to describe this better in the docs.